vmdb/product/toolbars/container_service_center_tb.yaml

```diff
@@ -16,9 +16,11 @@
       :text: 'Edit this #{ui_lookup(:table=>"container_service")}'
       :title: 'Edit this #{ui_lookup(:table=>"container_service")}'
       :url: '/edit'
+      :enabled: 'false'
     - :button: container_service_delete
       :image: delete
       :text: 'Remove this #{ui_lookup(:table=>"container_service")} from the VMDB'
       :title: 'Remove this #{ui_lookup(:table=>"container_service")} from the VMDB'
       :url_parms: '&refresh=y'
       :confirm: 'Warning: This #{ui_lookup(:table=>"container_service")} and ALL of its components will be permanently removed from the Virtual Management Database.  Are you sure you want to remove this #{ui_lookup(:table=>"container_service")}?'
+      :enabled: 'false'
```

Comment:
@h-kataria So, really, for all entities besides providers, we just want to completely remove the button.  I understand I would need to remove container_groups_center_tb.yaml (because nothing in there is being used) but I don't understand what you are talking about in the application_helper method. I am looking at the application_helper under apps/helper/application_helper but I don't see where I disable buttons from here.